README.rst

```diff
@@ -1158,6 +1158,10 @@ Choose from the list of available rules:
 
   A final class must not have final methods.
 
+* **no_unneeded_import_alias**
```

Comment:
looks like a candidate for our repo as well (make it part of `@PhpCsFixer` ruleset)

Reply:
ok